### PR TITLE
NFC: Wrapped runtime function TFE_Execute() with a new Swift runtime entrypoint

### PR DESCRIPTION
**Pull request by mhong**
"@_swift_tfc_EagerExecute()", which we will use to implement the new tracing
mode in future PRs.

Also advanced TF repo commit hash to bring in new experimental C APIs to handle
symbolic tensors and convert eager ops into graph nodes.

Added a few debug checks to TF stdlibs.

Turned off a few compiler/GPE tests that fail after bringing in TF repo
changes. They are tracked by https://bugs.swift.org/browse/SR-9736.

Also turned off a few GPU GPE runtime tests that hang after bringing in TF repo
changes. They are tracked by https://bugs.swift.org/browse/SR-9737.

### CODE REVIEW

**Comment by mhong** [comment]
@swift-ci please clean test tensorflow

**Comment by mhong** [comment]
Also CC @jekbradbury as FYI

File: stdlib/public/TensorFlow/CompilerRuntime.swift [hunk 1]
```diff
@@ -874,6 +874,21 @@ public extension _TensorComputation {
...
+@_cdecl("_swift_tfc_EagerExecute")
+func _TFCEagerExecute(_ op: CTFEOp,
+                      _ retvals: UnsafeMutablePointer<OpaquePointer?>,
+                      _ retvalCount: UnsafeMutablePointer<Int32>,
+                      _ status: CTFStatus) {
+  if _RuntimeConfig.printsDebugLog {
+    debugLog("Calling _TFCEagerExecute() over: ")
+    TFE_OpPrintDebugString(op)
+  }
+
```
Comment (rxwei) [inline stdlib/public/TensorFlow/CompilerRuntime.swift]: Extra line?

File: stdlib/public/TensorFlow/CompilerRuntime.swift [hunk 2]
```diff
@@ -874,6 +874,21 @@ public extension _TensorComputation {
...
+  TFE_Execute(op, retvals, retvalCount, status)
+}
+
+
```
Comment (rxwei) [inline stdlib/public/TensorFlow/CompilerRuntime.swift]: Same here

File: stdlib/public/TensorFlow/TensorHandle.swift [hunk 3]
```diff
@@ -88,6 +88,12 @@ public final class TensorHandle<Scalar> : _AnyTensorHandle
     self.init(copyingFromCTensor: cTensor)
     TF_DeleteTensor(cTensor)
   }
+
+  /// Return true if the underlying tensor is concrete (as opposed to being
+  /// symbolic).
+  public func isConcrete() -> Bool {
```
Comment (rxwei) [inline stdlib/public/TensorFlow/TensorHandle.swift]: Use a computed property ‘var isConcrete: Bool’.

**Review by rxwei — APPROVED** [review]

**Review by pschuh — APPROVED** [review]

File: stdlib/public/TensorFlow/CompilerRuntime.swift (hunk 1 above)
Comment (mhong) [reply stdlib/public/TensorFlow/CompilerRuntime.swift]: Removed

File: stdlib/public/TensorFlow/CompilerRuntime.swift (hunk 2 above)
Comment (mhong) [reply stdlib/public/TensorFlow/CompilerRuntime.swift]: Removed

File: stdlib/public/TensorFlow/TensorHandle.swift (hunk 3 above)
Comment (mhong) [reply stdlib/public/TensorFlow/TensorHandle.swift]: Nice suggestion. Done!

**Comment by mhong** [comment]
@swift-ci please clean test tensorflow

**Comment by mhong** [comment]
Linux tests passed. Mac testing had an error, but it does not appear related to this PR (@rxwei and I looked into this): https://ci-external.swift.org/view/Pull%20Request/job/swift-PR-TensorFlow-macOS/726/console

I'm submitting the PR to unblock sending out a subsequent PR for review.

**Pull request merged**